apps/financial-aid/web-osk/src/utils/taxCalculator.ts

```diff
@@ -0,0 +1,52 @@
+// import { AidDateAmountListItem } from './veitaFinancialAidDefinitions'
+
+export interface AidDateAmount {
+  personalTaxAllowance: number
+  taxPercentage: number
+}
+
+export const calculateAidFinalAmount = (
+  amount: number,
+  usePersonalTaxAllowance: boolean | undefined,
```

Comment:
This function should not take it undefined, the undefined should be unwrapped out side of this function.